docs/learn/learn-guides-treasury.md

```diff
@@ -0,0 +1,125 @@
+preimage with a treasury spend transaction is around
+{{ polkadot: 41 DOT :polkadot }}{{ kusama:  1.4 KSM  :kusama }}. Ensure you have enough account
+balance to pay for the submission deposit and the transaction fees.
+
+:::
+
+![Treasury Preimage](../assets/treasury/treasury-preimage.png)
+
+After successful submission of the preimage, it is displayed on Polkadot-JS UI > Governance >
+Preimages page. Every preimage is associated with a unique preimage hash (highlighted in a box in
+the image below). Take a note of this preimage hash, which is required to submit a referendum.
+
+![Treasury Preimage Hash](../assets/treasury/treasury-preimage-hash.png)
+
+### Submit a Treasury Track Referendum
+
+The example below shows how to submit a Treasury track referendum.
+
+- Navigate to [Polkadot-JS UI > Governance > Referenda](https://polkadot.js.org/apps/#/referenda)
+  and then click on Submit proposal.
+- Select the account which will be used to submit the proposal.
+- Choose the appropriate submission track (The example below selected Small Spender track).
+- Enter the preimage hash of the treasury spend transaction.(If the preimage exists on-chain, the
+  preimage length box is automatically populated)
+- Click on Submit proposal.
+- Sign and submit the transaction.
+
+![Submit Treasury Proposal](../assets/treasury/submit-proposal-treasury.png)
+
+Once your submission is executed, your referendum will appear under your chosen track on the
+Polkadot-JS UI [referenda page](https://polkadot.js.org/apps/#/referenda).
+
+### Place a Decision Deposit for the Treasury Track Referendum
+
+For the referendum to move from preparing phase to the deciding phase, a decision deposit needs to
+be placed. The decision deposit values for each individual
+[Treasury Tracks](./learn-polkadot-opengov-treasury#treasury-tracks) are listed in a section above
+in this document.
+
+![Submit Treasury Proposal Decision Deposit](../assets/treasury/treasury-proposal-decision-deposit.png)
+
+### Claiming the Preimage and Decision Deposits
+
+After the referendum finishes its life cycle (and gets approved or rejected or timed out), the
+deposits can be claimed. For claiming the preimage deposit, navigate to Polkadot-JS UI >
+Governance > Preimages and click on unnote button shown on the preimage you submitted. Similarly, to
+claim the decision deposit, navigate to Polkadot-JS UI > Governance > Referenda and scroll down to
+the end of the page to click on the referenda with the decision depoit and claim it.
+
+## Requesting Tipps from the Treasury
+
+To request a tipp funded by the treasury you can follow the above steps for
+[creating a treasury proposal](#creating-a-treasury-proposal) but instead of submitting the proposal
+to the `32 / Small Spender` track you will need to submit it to the `30 / Small Tipper` or
+`31 / Big Tipper` tracks depending on the number of tokens to be requested.
```

Comment:
```suggestion
To request a tip funded by the treasury, you can follow the above steps for
[creating a treasury proposal](#creating-a-treasury-proposal) but instead of submitting the proposal
to the `32 / Small Spender` track, you will need to submit it to the `30 / Small Tipper` or
`31 / Big Tipper` tracks depending on the number of tokens to be requested.
```